micro-ui/web/micro-ui-internals/packages/modules/workbench/src/configs/UICustomizations.js

```
@@ -809,5 +836,171 @@ export const UICustomizations = {
+      //to ADD SOR Composition
+      if (actionSelected === "ADD_SOR_COMPOSITION") {
+        window.location.href = `/works-ui/employee/rateanalysis/create-rate-analysis?sorid=${props?.uniqueIdentifier}`;
+      } else if (actionSelected === "VIEW_RATE_ANALYSIS") {
+        window.location.href = `/works-ui/employee/rateanalysis/view-rate-analysis?sorId=${
+          props?.masterName === "Composition" ? props?.data?.data?.sorId : props?.uniqueIdentifier
+        }&fromeffective=${props?.masterName === "Composition" ? props?.data?.data?.effectiveFrom : Date.now()}`;
+      }
+      //action===EDIT go to edit screen
+      else if (actionSelected === "EDIT") {
+        const additionalParamString = new URLSearchParams(props?.additionalParams).toString();
+        props?.history.push(`/${window?.contextPath}/employee/workbench/mdms-edit?moduleName=${props?.moduleName}&masterName=${props?.masterName}&uniqueIdentifier=${props?.uniqueIdentifier}${additionalParamString ? "&" + additionalParamString : ""}`)
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)